### PR TITLE
Konstantinivanov/sc 6790/as a devops i want to receive notification

### PR DESCRIPTION
**Pull request by konstantin-hero-labs**

### CODE REVIEW

File: .github/workflows/aws-deploy.yaml [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+        run: |
+          if [ "${{fromJson(steps.set_var.outputs.packageJson).services[0].runningCount }}" != "1" ]; then echo "SLACK_COLOR=#ff00ff" >> $GITHUB_ENV ; else echo "SLACK_COLOR=green" >> $GITHUB_ENV ; fi
+      - run: |
+          echo "${{ env.SLACK_COLOR }}"
+
+        # https://github.com/marketplace/actions/slack-notify
+      - name: Slack Notification
+        uses: rtCamp/action-slack-notify@v2
+        env:
+          SLACK_CHANNEL: Ivanov Konstantin
```
Comment (tomdos) [inline .github/workflows/aws-deploy.yaml:123]: maybe use env var to change it later ?

File: .github/workflows/aws-deploy.yaml [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+          SLACK_COLOR: ${{ env.SLACK_COLOR }} # or a specific color like 'good' or '#ff00ff'
+          SLACK_MESSAGE: ${{ github.event.head_commit.message }}
```
Comment (tomdos) [inline .github/workflows/aws-deploy.yaml:125]: Can you do it bit more user friendly with a mesasge like "Releasing XYZ to [environment] bla bla .. commit: 1234.."

File: .github/workflows/aws-deploy.yaml [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+          SLACK_TITLE: admin-dashboard
```
Comment (tomdos) [inline .github/workflows/aws-deploy.yaml:126]: maybe env var as well?

File: .github/workflows/aws-deploy.yaml [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+          SLACK_USERNAME: bot
+          SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
```
Comment (tomdos) [inline .github/workflows/aws-deploy.yaml:128]: use new lines pls ;-)

File: .github/workflows/aws-deploy.yaml [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+          content=`aws --region eu-west-2 ecs describe-services --services admin-dashboard --cluster hl-stage`
+          # the following lines are only required for multi line json
+          content="${content//'%'/'%25'}"
+          content="${content//$'\n'/'%0A'}"
+          content="${content//$'\r'/'%0D'}"
+          # end of optional handling for multi line json
+          echo "::set-output name=packageJson::$content"
+      - id: status
+        run: |
+          if [ "${{fromJson(steps.set_var.outputs.packageJson).services[0].runningCount }}" != "1" ]; then echo "SLACK_COLOR=#ff00ff" >> $GITHUB_ENV ; else echo "SLACK_COLOR=green" >> $GITHUB_ENV ; fi
```
Comment (tomdos) [inline .github/workflows/aws-deploy.yaml:115]: Is this a color for success / failure ?

**Review by tomdos — APPROVED** [review]
LGTM. there are a few minors as it is specific to admin-dashboard etc but this might be address later if you wish

File: .github/workflows/aws-deploy.yaml (hunk 5 above)
Comment (konstantin-hero-labs) [reply .github/workflows/aws-deploy.yaml:115]: Yes

File: .github/workflows/aws-deploy.yaml (hunk 2 above)
Comment (konstantin-hero-labs) [reply .github/workflows/aws-deploy.yaml:125]: Screenshot from 2022-05-06 16-52-46

This form already contains some text, maybe it is enough

Comment (tomdos) [reply .github/workflows/aws-deploy.yaml:131]: oh, that looks goood

**Review by tomdos — APPROVED** [review]
Thanks

**Pull request merged**